deploytools.php

```diff
@@ -68,8 +68,18 @@ public function onBeforeCompileHead() {
 			}  
 			$head['scripts'] = $newarray;
 		}
+		if ($this->params->get('process_custom', 1)) {
+			$newarray = array();
+			foreach ($headerstuff['custom'] as $key => $value) {
+				$appendjs = ".js?".$version;
+				$appendcss = ".css?".$version;
+				$outputjs = str_replace(".js",$appendjs,$value);
+				$newarray[$str] = str_replace(".css",$appendcss,$outputjs);
+			} 
```

Comment:
- PHPCS missing
- Improve code readability. See if you can reduce use of variables